src/backend/controller/user/user.ts

```
@@ -3,28 +3,120 @@ import {
   insertUser,
   updateUser,
   deleteUser,
+  payOutUser,
 } from '@backend/services/db/user/user_db';
 import type { NextApiRequest, NextApiResponse } from 'next';
 import { Request } from '@backend/middleware/verify_request';
 
 export const getUserController = async (req: Request, res: NextApiResponse) => {
-  getUser(req, res);
+  try {
+    // break down request <===========
+    const { email } = req.user;
+    // call service <===============
+    const userData = await getUser(email);
```

the service method should throw an exception if user is not found

---

Thanks, fixed it!